packages/desktop/App.svelte

```diff
@@ -112,15 +112,21 @@
         })
         Electron.onEvent('menu-create-developer-profile', () => {
             get(appRouter).reset()
-            initialiseOnboardingProfile(true, NetworkProtocol.Shimmer)
-            initialiseOnboardingRouters()
+            void initialiseOnboarding({
+                isDeveloperProfile: true,
+                networkProtocol: NetworkProtocol.Shimmer,
+                resetRouters: true,
+            })
             get(appRouter).next({ shouldAddProfile: true })
```

Comment:
We should explicitly state what route to go to in the initialiseOnboarding function